unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/LoadingHUD/Bridge/LoadingBridge.cs

```diff
@@ -26,4 +36,33 @@ public void SetLoadingScreen(string jsonMessage)
             DataStore.i.HUDs.loadingHUD.message.Set(payload.message);
         DataStore.i.HUDs.loadingHUD.showTips.Set(payload.showTips);
     }
+    
+    public void FadeInLoadingHUD(string jsonMessage)
+    {
+        StartCoroutine(WaitForLoadingHUDVisible(jsonMessage));
+    }
+    
+    IEnumerator WaitForLoadingHUDVisible(string jsonMessage)
```

Comment:
I'm not sure this is the place for this logic. Bridges should have no logic whatsoever, they should just put info in the datastore or call stuff.

In any case, this is temporary until we own full control of the loading flow in the renderer so we can let it pass through. In any case let's add comments and TODOs.